protocol/interfaces.go

```diff
@@ -5,13 +5,22 @@ import (
 	"math/big"
 )
 
+// ChainStatusInfo represents chain status with selector, block height and disabled state.
+type ChainStatusInfo struct {
+	ChainSelector ChainSelector
+	BlockHeight   *big.Int
```

Comment:
Can we stick to a single term to refer to a block number? its BlockHeight here, its blockNumber in a lot of other places.